test/functional/apps/context/_context_navigation.ts

```diff
@@ -76,7 +76,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           await PageObjects.discover.waitForDocTableLoadingComplete();
 
           for (const [columnName, value] of TEST_FILTER_COLUMN_NAMES) {
-            expect(await filterBar.hasFilter(columnName, value)).to.eql(true);
+            expect(await filterBar.hasFilter(columnName, value, false)).to.eql(true);
```

Comment:
What caused these tests to change? I can't quite figure it out 😅

Reply:
Before: define filter, open context (filter getting disabled), go back via breadcrumb to main-> prev filters shown (enabled).
After: define filter, open context (filter getting disabled), go back via breadcrumb to main-> prev filters shown (disabled).
but, also After: new filters applied in context will be shown when go back via breadcrumb to main. With that fact I've decided not to change the state of filters when go back via breadcrumb to main. Maybe we also don't need to disable them when go to context. WDYT? @kertal

Reply:
When going to back to main, we should not add any additional filters or change the state of them in the main view. The way I see surrounding documents is, it has local state, that can be changed, like the filters. But it should not influence the main state.

Reply:
I think @kertal's explanation is what the team decided on last time we discussed it. We also have an issue to track some of the problems with filters in Discover that explores some of this: #136316. Most the updates to filter handling in this PR are great though, it's just this change might be an issue.

Reply:
Reverted.